SU-HSDO/stanford_mrc
# Mrcd8 204 basic page changes

# READY FOR REVIEW


# Summary
- Basic page changes except for quick edit on bricks

# Needed By (Date)
- soon
# Urgency
- end of sprint

# Steps to Test

1. sync db with prod
2. check out this branch and drush updb
3. Make sure line items in MRCD8-204 are working as expected except for inline editing.

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-204
